CHANGELOG_iOS.md

```diff
@@ -1,5 +1,38 @@
+ - Added support for Dark Mode and Tinted icons. ([#41088](https://github.com/brave/brave-browser/issues/41088))
+ - Added Leo to the Home Screen shortcuts widget. ([#39886](https://github.com/brave/brave-browser/issues/39886))
+ - Added support for modifying user prompt in Leo. ([#39831](https://github.com/brave/brave-browser/issues/39831))
+ - Added skipping debouncing if canonicalized URL’s hostname doesn’t match unescaped URLs hostname. ([#39866](https://github.com/brave/brave-browser/issues/39866))
+ - Added ability to edit custom search engines after being added in ‘Search Engines’ settings page. ([#39552](https://github.com/brave/brave-browser/issues/39552))
+ - Added DarkReader support when Night Mode is enabled. ([#39786](https://github.com/brave/brave-browser/issues/39786))
+ - Added handling potential infinite redirect loop with HTTPS upgrades. ([#40346](https://github.com/brave/brave-browser/issues/40346))
+ - Added right-to-left (RTL) language support. ([#40471](https://github.com/brave/brave-browser/issues/40471))
+ - Updated ’Manage Website Data’ to remove ads data as well when clearing browsing history. ([#39051](https://github.com/brave/brave-browser/issues/39051))
+ - Updated icon for starting a new conversation in Leo. ([#39818](https://github.com/brave/brave-browser/issues/39818))
+ - Updated input box to be in focus when Leo in opened. ([#39858](https://github.com/brave/brave-browser/issues/39858))
+ - Updated copy in Brave Rewards panel for non-connected users. ([#40909](https://github.com/brave/brave-browser/issues/40909))
+ - Fixed crash when launching Brave via Home Screen shortcuts widget search button. ([#41457](https://github.com/brave/brave-browser/issues/41457))
+ - Fixed crash when tapping on search icon to open search settings after opening URL bar. ([#39136](https://github.com/brave/brave-browser/issues/39136))
+ - Fixed crash occurring for some users when loading internal pages. ([#41445](https://github.com/brave/brave-browser/issues/41445))
+ - Fixed crash when pop-up is presented for Japan users. ([#41443](https://github.com/brave/brave-browser/issues/41443))
+ - Fixed crash in Playlist when saving data offline with internet connection disabled. ([#41388](https://github.com/brave/brave-browser/issues/41388))
+- Fixed bookmark count to show correctly when adding ‘Saved Tabs’ via tab icon. ([#38340](https://github.com/brave/brave-browser/issues/38340))
+ - Fixed not being able to open links from X app using ‘Open in Brave’ from share menu. ([#41296](https://github.com/brave/brave-browser/issues/41296))
+ - Fixed security issue with URL formatting as reported in HackerOne by renwa. ([#41716](https://github.com/brave/brave-browser/issues/41716))
+ - Fixed security issue involving interstitial pages not showing deceptive site warning as reported in HackerOne by renwa. ([#41803](https://github.com/brave/brave-browser/issues/41803))
```

Comment:
Should be formatted with `[Security]`. Example:

```
 - [Security] Fixed issue involving interstitial pages not showing deceptive site warning as reported on HackerOne by renwa. ([#41803](https://github.com/brave/brave-browser/issues/41803))
```

Basically what we do with `Android` & `Desktop` as well. Please move this at the top of `Fixed` as it's a security issue.